i18n/dummy.py

```diff
@@ -138,18 +138,24 @@ class Dummy(BaseDummyConverter):
         """.split()
     )
 
-    # To simulate more verbose languages (like German), pad the length of a string
-    # by a multiple of PAD_FACTOR
-    PAD_FACTOR = 1.33
-
     def pad(self, string):
-        """add some lorem ipsum text to the end of string"""
+        """
+        Add some lorem ipsum text to the end of string to simulate more verbose languages (like German).
+        Padding factor determined by http://www.w3.org/International/articles/article-text-size.en
+        """
         size = len(string)
-        if size < 7:
+        if size <= 10:
             target = size * 3
+        elif size <= 20:
+            target = size * 2
```

Comment:
I did look at the length, and it's fuzzy, because it's an "average" expansion, with a range.  Also, the table in the link has  dip: 31-50 goes up to 160%, but 51-70 goes up to 170%? That looks like a typo to me.  So we have a lot of leeway in how to use this data.  I don't like that 11 chars comes out shorter than 10 chars. :(